arbnode/mel/delayed_message_backlog.go

```diff
@@ -9,7 +9,7 @@ type DelayedMeta struct {
 	MelStateParentChainBlockNum uint64
 }
 
-// DelayedMetaBacklog is a data structure that holds a deque containing meta data related to delayed messages that are currently SEEN by MEL
+// DelayedMessageBacklog is a data structure that holds a deque containing meta data related to delayed messages that are currently SEEN by MEL
 // but not yet READ. This enables verification of correctness of delayed messages read from the database, i.e each MEL state has the
 // DelayedMessageMerklePartials field that holds merkle partials array after all the delayed messages SEEN while constructing the state are
```

Comment:
Because these are used for verification, we can call them Merkle witnesses, something like DelayedMessageTreeWitnesses